### PR TITLE
feat(clickhouse): Upgrade to 24.8

### PR DESCRIPTION
**Pull request by biggerfisch**
### Summary

Upgrades the ClickHouse recipe to 24.8, to match the ClickHouse Cloud release cadence.

https://clickhouse.com/docs/en/changelogs/24.8

### CODE REVIEW

**Comment by biggerfisch** [comment]
* **#85** <a href="https://app.graphite.dev/github/pr/memfault/conda-recipes/85?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @biggerfisch and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: clickhouse/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-  - url: https://github.com/ClickHouse/ClickHouse/releases/download/v{{ version }}-stable/clickhouse-macos-aarch64 # [osx and arm64]
-    sha256: 2ac1ad4b2a66c84634f569368d40bb26a1c1c5b5d252bb8f827a8acf7203d889 # [osx and arm64]
-  - url: https://github.com/ClickHouse/ClickHouse/releases/download/v{{ version }}-stable/clickhouse-common-static-{{ version }}-amd64.tgz # [linux64]
-    sha256: 1628abfe7275488d3b7b192d8ddba2872d470077d6f4fca884a0941509e57867 # [linux64]
+  - url: https://github.com/ClickHouse/ClickHouse/releases/download/v{{ version }}-lts/clickhouse-macos # [osx and x86_64]
+    sha256: f2b90c95b52a9184e2ae39ca8b5be56cdefaea0ffe3c13fa81d916d828c1758d # [osx and x86_64]
+  - url: https://github.com/ClickHouse/ClickHouse/releases/download/v{{ version }}-lts/clickhouse-macos-aarch64 # [osx and arm64]
+    sha256: 3354d8f58f87bb9182c75ad6c4d71d8da97fd16b16449f7a61988d2b44640686 # [osx and arm64]
+  - url: https://github.com/ClickHouse/ClickHouse/releases/download/v{{ version }}-lts/clickhouse-common-static-{{ version }}-amd64.tgz # [linux64]
+    sha256: dae03e7ade008470bf764e5f7d8a1d657ba1c8b9f62230d7afcbe3d03576169a # [linux64]
```
Comment (fnune) [inline clickhouse/meta.yaml:14]: Is changing the channel OK? From `stable` to `lts`. It's our own thing so :shrug: but does this set a precedent that we'll only take future `lts` releases and not `stable` releases?

**Review by fnune — APPROVED** [review]

File: clickhouse/meta.yaml (hunk 1 above)
Comment (biggerfisch) [reply clickhouse/meta.yaml:14]: No, we change this back and forth every time haha - it's just due to a hacky condo recipe

**Comment by biggerfisch** [comment]
### Merge activity

* **Oct 18, 5:39 AM EDT**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/memfault/conda-recipes/85).

**Pull request merged**